source/agora/test/Base.d

```diff
@@ -447,15 +447,25 @@ extern(D):
     /// number of txs required for nomination
     protected ulong txs_to_nominate;
 
+    /// test start time
+    protected ulong test_start_time;
```

Comment:
Nitpick, but why not just `start_time` ? We know this refers to the tests for sure.

Reply:
we also have 'initial_time' which is alone conusing enough I think:

```D
/// The initial clock time of every spawned node. Note that if there were
/// any extra blocks loaded (`blocks` in the ctor) then the initial time
/// will be test_start_time + (last_height * block_interval)
protected time_t initial_time;
```

Reply:
Fair point, ignore then.